### DISCUSSION TITLE
Login page change gopher Icon

### DISCUSSION THREAD

**Discussion by NeptuneReaper** [seriousm4x/UpSnap#758] [Help wanted] [answered]
Hello, I am configuring Upsnap for my company and I found out that , after changed the favicon for the personal use i found out that the login Page icon keeps having the Gopher.
I tried to scroll to any other settings but I couldn't find how to change it.
Is there a way how can i change it? 

Thank you.

**Comment by seriousm4x** [comment, marked as answer]
I've just added it, thanks! https://github.com/seriousm4x/UpSnap/commit/fb5ef85a74597253baa5c096ab62e106fb2aa4cc

**Comment by NeptuneReaper** [comment]
> I've just added it, thanks! https://github.com/seriousm4x/UpSnap/commit/fb5ef85a74597253baa5c096ab62e106fb2aa4cc

Will it be an option in the next beta?

Thank you very much

**Reply by seriousm4x** [reply]
It will automatically be the same as in the navbar or tab

**Reply by NeptuneReaper** [reply]
Thank you very much for the update
